### PR TITLE
Clean up schedules and tests (Part 2/2)

### PR DESCRIPTION
**Pull request by csarofeen**
Cleaning up the APIs of the schedulers and fixing the tests associated with them:
- [x]   None,
- [x]   NoOp,
- [x]   PointWise,
- [x]   Matmul,
- [x]   Reduction,
- [x]   InnerPersistent,
- [x]   InnerOuterPersistent,
- [x]   OuterPersistent,
- [x]   Transpose,
- [x]   ExprEval

### CODE REVIEW

**Comment by csarofeen** [comment]
!build

File: csrc/scheduler/matmul_utils.cpp
Comment (naoyam) [inline csrc/scheduler/matmul_utils.cpp:1]: @jacobhinkle Could you review this file?

**Review by naoyam — COMMENTED** [review]
Looks good. Just wanted to have another eye on the matmul scheduler.

File: csrc/scheduler/matmul_utils.cpp [hunk 1]
```diff
@@ -39,6 +39,7 @@
 #include "utils.h"
 
 namespace nvfuser {
+namespace matmul_utils {
```
Comment (jacobhinkle) [inline csrc/scheduler/matmul_utils.cpp:42]: Good idea. As a separate issue, we should think about combining or more clearly naming the difference between `mma_utils` and `matmul_utils`.

**Review by jacobhinkle — APPROVED** [review]
Changes to `matmul_utils.cpp` look good to me

**Pull request merged**